### ISSUE TITLE
Question about XBOX renderer setting

### ISSUE THREAD

**Issue by Sami32** [UniversalMediaServer/UniversalMediaServer#1565]
@SubJunk Can you remember why this limitation has been set?
`TranscodedVideoAudioSampleRate = 44100`
I didn't see any XBOX 360 specifications related to that so i guess that it is related to a very specific case that you have met?

**Comment by SubJunk** [comment]
Because without it, audio in transcoded videos didn't work on Xbox 360
Done in https://github.com/UniversalMediaServer/UniversalMediaServer/commit/ecbc33fe615ff4d3d72c57a81dbe0e4fee56d137

**Comment by Sami32** [comment]
It has been done long before 2011 in PMS but i cannot find any such report or specifications related to that.

**Comment by OP-Klaus** [comment]
Yeah we tested it with actual Xboxes. Specs/docs are often wrong :)

**Comment by SubJunk** [comment]
Yeah we tested it with actual Xboxes. Specs/docs are often wrong :)

**Comment by SubJunk** [comment]
I don't remember
